pom.xml

```diff
@@ -107,4 +107,4 @@
 		</plugins>
 	</build>
 
-</project>
+</project>
```

Comment:
What was the purpose of these edits in pom.xml @dibbor ?
It looks like nothing was changed, just deleted then re-added

Reply:
I was having problems building it initially so I googled and tried stuff but later just copy pasted the original one.